test/e2e/app-dir/app/index.test.ts

```diff
@@ -749,6 +749,28 @@ createNextDescribe(
             '{"category":"books","id":"hello-world"}'
           )
         })
+
+        it('should allow dynamic routes to access cookies', async () => {
```

Comment:
Can we throw this in the `app-static` suite to keep checks like this there?

Reply:
Would be nice to also assert that we still error in `dev` correctly if dynamic usage is done with `dynamic = 'error'`